groups/ecs/sg.tf

```diff
@@ -0,0 +1,37 @@
+resource "aws_security_group" "alb_security_group" {
+  name        = "${var.environment}-${var.service}-lb"
+  description = "Restricts access for ${var.service}-${var.environment} lb artifactory nodes"
+  vpc_id      = data.aws_vpc.placement.id
+
+  ingress {
+    description     = "lb HTTP ingress from admin and concourse CIDRs"
+    from_port       = 80
+    to_port         = 80
+    protocol        = "tcp"
+    cidr_blocks     = local.placement_subnet_cidrs
+    prefix_list_ids = [data.aws_ec2_managed_prefix_list.administration.id]
+  }
+
+  ingress {
+    description     = "lb HTTPS ingress from admin and concourse CIDRs"
+    from_port       = 443
+    to_port         = 443
+    protocol        = "tcp"
+    cidr_blocks     = local.placement_subnet_cidrs
+    prefix_list_ids = [data.aws_ec2_managed_prefix_list.administration.id]
+  }
```

Comment:
Concourse doesn't need to access Grafana via HTTP/HTTPS. Also you're referencing `local.placement_subnet_cidrs` which would appear to be the subnets that the ECS cluster is being made avaiulable on - so you're allowing it to connect to itself? That isn't necessary.